LSP-typescript.sublime-settings

```diff
@@ -2,40 +2,31 @@
 	"languages": [
 		{
 			"languageId": "javascriptreact",
-			"scopes": [
-				"source.jsx",
-				"source.js.react",
-			],
+			"scopes": ["source.jsx", "source.js.react"],
 			"syntaxes": [
 				"Packages/User/JS Custom/Syntaxes/React.sublime-syntax",
 			],
 		},
 		{
 			"languageId": "javascript",
-			"scopes": [
-				"source.js",
-			],
+			"scopes": ["source.js"],
 			"syntaxes": [
 				"Packages/JavaScript/JavaScript.sublime-syntax",
 				"Packages/Babel/JavaScript (Babel).sublime-syntax",
 			],
 		},
 		{
-			"languageId": "typescript",
-			"scopes": [
-				"source.ts",
-			],
+			"languageId": "typescriptreact",
+			"scopes": ["source.tsx", "source.ts.react"],
```

Comment:
`source.ts.react` 
Where did you get this scope? :)

Reply:
made it up. Seems consistent with source.js.react